### PR TITLE
[Synapse Managemenet] Support IntegrationRuntime, FirewallRule and Vnet features

### PR DESCRIPTION
**Pull request by aim-for-better**
1. Fix Update SqlPool issue
2. Support IntegrationRuntime feature
3. Support Get single Firewall Rule
4. Support Vnet feature

### CODE REVIEW

File: sdk/synapse/Microsoft.Azure.Management.Synapse/src/Microsoft.Azure.Management.Synapse.csproj [hunk 1]
```diff
@@ -7,11 +7,14 @@
 		<PackageId>Microsoft.Azure.Management.Synapse</PackageId>
 		<Description>Azure Synapse Management SDK Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Synapse</AssemblyName>
-		<Version>0.1.0-preview.1</Version>
+		<Version>0.2.0-preview</Version>
```
Comment (aim-for-better) [inline sdk/synapse/Microsoft.Azure.Management.Synapse/src/Microsoft.Azure.Management.Synapse.csproj:10]: Is 0.2.0-preview right here?  Should it be `0.1.0-preview.2` or `0.2.0-preview`?
This PR adds some api.

Comment (isra-fel) [reply sdk/synapse/Microsoft.Azure.Management.Synapse/src/Microsoft.Azure.Management.Synapse.csproj:10]: `0.1.0-preview.2`. And when you decide it's stable, `0.1.0`.
And then `0.2.0-preview.1`, and then `0.2.0`

Comment (aim-for-better) [reply sdk/synapse/Microsoft.Azure.Management.Synapse/src/Microsoft.Azure.Management.Synapse.csproj]: > Is 0.2.0-preview right here? Should the version be `0.1.0-preview.2` or `0.2.0-preview`?
> If the version is 0.2.0-preview, can we set the version to 0.1.0 when we finish the preview phase?

Thanks for clarification. I have updated it to 0.1.0-preview.2.

**Review by isra-fel — CHANGES_REQUESTED** [review]
Please write tests to properly cover new operations. Thanks

**Comment by aim-for-better** [comment]
> Please write tests to properly cover new operations. Thanks

Hi Yeming, Now we don't have test project

**Review by isra-fel — APPROVED** [review]
Service hasn't GA. Will skip the tests until then.

**Pull request merged**